packages/server/src/tests/banners/bannerDeployTimes.ts

```diff
@@ -47,15 +47,31 @@ interface BannerDeployTimes {
     RestOfWorld: Date;
     EuropeanUnion: Date;
 }
-export interface BannerDeployCaches {
-    contributions: () => Promise<BannerDeployTimes>;
-    subscriptions: () => Promise<BannerDeployTimes>;
+
+interface BannerDeployTimesProviders {
+    contributions: ValueProvider<BannerDeployTimes>;
+    subscriptions: ValueProvider<BannerDeployTimes>;
 }
 
-const cachedDeployTime = (bannerChannel: BannerChannel): (() => Promise<BannerDeployTimes>) =>
-    cacheAsync(fetchBannerDeployTimes(bannerChannel), { ttlSec: fiveMinutes, warm: true });
+export class BannerDeployTimesProvider {
+    providers: BannerDeployTimesProviders;
+
+    constructor(providers: BannerDeployTimesProviders) {
+        this.providers = providers;
+    }
+
+    getDeployTimes(channel: 'contributions' | 'subscriptions'): BannerDeployTimes {
```

Comment:
Could we use `BannerChannel` here?

Reply:
Unfortunately `BannerChannel` also includes `signIn`, which doesn't have redeploys

Reply:
Ah okay that makes sense, thanks 👍

Reply:
hopefully one day we'll combine these into one channel, or at least replace them with better names!